src/preamble.js

```diff
@@ -1492,17 +1493,6 @@ HEAP16[1] = 0x6373;
 if (HEAPU8[2] !== 0x73 || HEAPU8[3] !== 0x63) throw 'Runtime error: expected the system to be little-endian!';
 #endif
 
-Module['HEAP'] = HEAP;
-Module['buffer'] = buffer;
-Module['HEAP8'] = HEAP8;
-Module['HEAP16'] = HEAP16;
-Module['HEAP32'] = HEAP32;
-Module['HEAPU8'] = HEAPU8;
-Module['HEAPU16'] = HEAPU16;
-Module['HEAPU32'] = HEAPU32;
-Module['HEAPF32'] = HEAPF32;
-Module['HEAPF64'] = HEAPF64;
-
```

Comment:
Does this mean we don't export `HEAP*` anymore? We don't need these documentations then? http://kripken.github.io/emscripten-site/docs/api_reference/preamble.js.html#type-accessors-for-the-memory-model

Reply:
We still do, this was just done twice ;) It's still done here: https://github.com/kripken/emscripten/blob/10c9c4a3bf3978ff7b1fba7b372635618a1c1fe8/src/preamble.js#L885